app/views/ctc/questions/confirm_legal/edit.html.erb

```diff
@@ -9,6 +9,8 @@
   <%= form_with model: @form, url: current_path, local: true, method: "put", builder: VitaMinFormBuilder, data: { efile_security_information: true, form_name: form_name }, html: { class: "form-card" } do |f| %>
     <%= f.cfa_checkbox(:consented_to_legal, t("views.ctc.questions.confirm_legal.consent"), options: { checked_value: "yes", unchecked_value: "no" }) %>
     <%= recaptcha_v3(action: 'confirm_legal') %>
-    <%= f.submit t("views.ctc.questions.confirm_legal.action"), class: "button button--primary button--wide text--centered spacing-above-60", data: { disable_with: t("general.please_wait") } %>
+    <div class="spacing-above-60">
+      <%= f.continue t("views.ctc.questions.confirm_legal.action") %>
```

Comment:
FYI to reviewers: This is a side-effect of the general strategy of leaving the text in the button the same when it gets disabled. We didn't specifically quiz Anu about this but since Anu expressed a strong desire for consistency, we feel good about this change.